service/controller/resource/app/desired.go

```diff
@@ -6,12 +6,14 @@ import (
 	"strconv"
 
 	g8sv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/clusterclient/service/release/searcher"
 	"github.com/giantswarm/microerror"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	apiv1alpha2 "sigs.k8s.io/cluster-api/api/v1alpha2"
 
 	"github.com/giantswarm/cluster-operator/pkg/annotation"
+	appspec "github.com/giantswarm/cluster-operator/pkg/app"
```

Comment:
```suggestion
	pkgapp "github.com/giantswarm/cluster-operator/pkg/app"
```

Reply:
This package had been deleted so I will mark this as resolved.